Filter/Basic.php

```diff
@@ -203,13 +203,13 @@ public function getParameters()
 
     /**
      * Get the stream resource being filtered.
-     * Maybe available only during filter calls when the closing parameter is
-     * set to false.
+     * Maybe available only during `static::filter` calls when the closing
+     * parameter is set to false.
      *
      * @return  resource
      */
     public function getStream()
     {
-        return $this->stream;
+        return isset($this->stream) ? $this->stream : null;
```

Comment:
This attribute seems to be always set. We should test the value, but this is not really interesting to support another value in case of… in case of what?

My guess is: It supports with current PHP implementation. Stream filters are purely awesome but not very well-documentented neither very well-defined. We must not change anything before having a strong test suite.

Thoughts?